conference/cfp.py

```diff
@@ -103,6 +103,7 @@ def submit_proposal_step2_add_speakers(request, talk_uuid):
                 speaker = save_information_from_speaker_form(
                     request.user, speaker_form.cleaned_data
                 )
+                print(speaker_form.cleaned_data)
```

Comment:
Is this for debugging purposes? Might be good to remove this.

Reply:
Yeah it was for debugging, thanks for spotting it, removed it now